brightlayer-ui/react-native-workflows
# BLUI-2966 Fix keyboard next button with refs

Fixes #73 & BLUI-2966.


#### Changes proposed in this Pull Request:
- Fix keyboard next button for navigating between default account details and custom account details


#### To Test:
- `git clone git@github.com:brightlayer-ui/react-native-workflows.git -b feature/blui-2966-fix-keyboard-next-button-with-refs`
- `cd react-native-workflows/login-workflow`
- `yarn initialize`
- `cd shared-auth && git checkout feature/blui-2966-fix-keyboard-next-button-with-refs`
- `cd .. && yarn start:example`
- check both "Invite Register" and "Self Register" flows to ensure that you can navigate between pages and form fields using the native keyboard next buttons.

Note: There's a weird bounce that happens when moving to the 3rd and 4th input fields. I've checked this and it's an issue across the board if the form extends below the keyboard. If we want this fixed specifically it will need addressed in another story

Disclaimer: I need to update some docs around this, but want to make sure this method works first before documenting it.
